bgpd/bgp_open.h

```diff
@@ -45,8 +44,7 @@ struct capability_orf_entry
     u_char type;
     u_char mode;
   } orfs[];
-} __attribute__ ((packed));
-#pragma pack()
```

Comment:
Why are we removing the pack:sharpd@robot ~/frr> more foo.c
#include <stdio.h>

typedef unsigned char safi_t;

typedef enum {
  IANA_AFI_RESERVED = 0,
  IANA_AFI_IPV4 = 1,
  IANA_AFI_IPV6 = 2,
  IANA_AFI_L2VPN = 25,
  IANA_AFI_IPMR = 128,
  IANA_AFI_IP6MR = 129
} iana_afi_t;

/* Generic MP capability data */
struct capability_mp_data
{
  iana_afi_t afi;
  unsigned char reserved;
  safi_t safi;
};

struct capability_orf_entry 
{
  struct capability_mp_data mpc;
  unsigned char num;
  struct {
    unsigned char type;
    unsigned char mode;
  } orfs[];
};

#pragma pack(1)
struct pcapability_orf_entry
{
  struct capability_mp_data mpc;
  unsigned char num;
  struct {
    unsigned char type;
    unsigned char mode;
  } orfs[];
} __attribute__ ((packed));
#pragma pack()

void main (void)
{
   printf ("Non packed: %d\n" ,sizeof( struct capability_orf_entry));
   printf ("Packed: %d\n", sizeof (struct pcapability_orf_entry));
}
sharpd@robot ~/frr> ./foo
Non packed: 12
Packed: 9
sharpd@robot ~/frr> 

Plus orfs is used as a variable sized array.  What am I missing?